docusaurus/docs/reactnative/common-content/core-components/channel-list/props/load_more_threshold.mdx

```diff
@@ -1,5 +1,5 @@
-Sets the [`onEndReachedThreshold`](https://reactnative.dev/docs/flatlist#onendreachedthreshold) of the underlying FlatList.
+Sets the [`onEndReachedThreshold`](https://reactnative.dev/docs/flatlist#onendreachedthreshold) of the underlying FlatList. We recommend using `0.1` as the default value for this prop as changing it might hit additional `channelQuery` calls and you might hit the limit.
```

Comment:
```suggestion
Sets the [`onEndReachedThreshold`](https://reactnative.dev/docs/flatlist#onendreachedthreshold) of the underlying FlatList. We recommend using `0.1` as the default value for this prop, as changing it might hit additional `channelQuery` calls, and you might reach the limit.
```

Reply:
2 times hit so I've changed it.